### PR TITLE
Added ip field to ClubVisit model

### PR DESCRIPTION
**Pull request by cphalen**
Added IP address to the ClubVisit model and updated serializer to save IP address. All existing ClubVisit entires are defaulted to have `null` IP.

[ch2980]

### CODE REVIEW

File: backend/clubs/models.py
Comment (ezwang) [inline backend/clubs/models.py:765]: Instead of a `CharField`, there is a [GenericIPAddressField](https://docs.djangoproject.com/en/3.1/ref/models/fields/#genericipaddressfield) that you can use.

File: backend/clubs/serializers.py
Comment (ezwang) [inline backend/clubs/serializers.py:1415]: This won't actually do anything to save the IP into the model, it'll just return the user their own IP in the response. You'll need to save the IP when you save the `ClubVisit` object, not when you fetch it.

**Review by ezwang — CHANGES_REQUESTED** [review]
Left a few comments on the PR. Should be good to merge after you fix them!

File: backend/clubs/serializers.py
Comment (cphalen) [reply backend/clubs/serializers.py:1415]: Ok great point not sure how I didn't notice this earlier

File: backend/clubs/models.py
Comment (cphalen) [reply backend/clubs/models.py:765]: Thanks!

File: backend/clubs/serializers.py
Comment (ezwang) [inline backend/clubs/serializers.py:1416]: You can remove this if statement, you don't want to save the IP if the user specifies it (always trust the server, no reason for user to specify it).

Comment (cphalen) [reply backend/clubs/serializers.py:1416]: Ok will do!

File: backend/clubs/migrations/0068_clubvisit_ip.py
Comment (ezwang) [inline backend/clubs/migrations/0068_clubvisit_ip.py:9]: You'll need to delete and regenerate this migration after Lucy merged in her PR, since you've both made database edits.

File: backend/clubs/serializers.py
Comment (ezwang) [inline backend/clubs/serializers.py:1428]: I'm pretty sure you can leave out the `"ip"` and the HiddenField altogether, and it will do the same thing. You'll need to test this, though.

File: backend/clubs/migrations/0068_clubvisit_ip.py
Comment (ezwang) [reply backend/clubs/migrations/0068_clubvisit_ip.py:9]: Also, delete the `backend/.swp` file in this PR.

File: backend/clubs/serializers.py
Comment (cphalen) [reply backend/clubs/serializers.py:1431]: I double checked this and yes it did the same thing! Thanks for pointing that out

**Review by ezwang — APPROVED** [review]
LGTM!

**Pull request merged**
